pkg/pool-cow/contracts/CowRouter.sol

```diff
@@ -145,6 +167,23 @@ contract CowRouter is SingletonAuthentication, VaultGuard, ICowRouter {
         );
     }
 
+    /********************************************************
+                     Withdraw Protocol Fees
+    ********************************************************/
+    /// @inheritdoc ICowRouter
+    function withdrawProtocolFees(IERC20 token) external {
+        uint256 amountToWithdraw = _protocolFees[token];
+        if (amountToWithdraw == 0) {
```

Comment:
Do we really need this? Would be simpler to not revert, and simply have: if(amountToWithdraw > 0) { <code> } and let it fall through and do nothing in the zero balance case.